docs/03.API-EXAMPLE.md

```diff
@@ -1,8 +1,42 @@
 JerryScript Engine can be embedded into any application, providing the way to run JavaScript in a large range of environments - from desktops to low-memory microcontrollers.
 
-This guide is intended to introduce you to JerryScript embedding API through creation of simple JavaScript shell.
+This guide is intended to introduce you to JerryScript embedding API and to create a minimal JavaScript shell.
+The examples are not using all API methods please also check out the API reference document which contains additional examples.
 
-## Step 1. Execute JavaScript from your application
+
+## Before trying out the examples: Get and build JerryScript library
+
+Before getting started using the JerryScript library it should be cloned and built for a target os/device.
+
+There are quite a few configuration options but for these examples the JerryScript is built
+**with default configuration** and installed to a user directory on a Linux system.
+This is done by the following commands:
+
+```sh
+$ mkdir jerry
```

Comment:
I'm not sure but based on the build commands below I think you meant `mkdir jerry && cd jerry` here.